client/packages/programs/src/JsonForms/common/components/Autocomplete.tsx

```diff
@@ -52,7 +52,15 @@ const UIComponent = (props: ControlProps) => {
     () => getDisplayOptions(schemaOptions),
     [schemaOptions]
   );
+
+  const clearable = !props.config?.required;
+
   useEffect(() => {
+    if (!data && clearable) {
```

Comment:
is clearable always available (are params and config available from the start) ? need to be in dependencies ?

Reply:
I think it was ok, bc if clearable is false, can't set data to be undefined anyway, but good catch thanks have added 🙏

Reply:
[23a5727](https://github.com/msupply-foundation/open-msupply/pull/6678/commits/23a5727f0757512e8ad51cf1821a3c51bf840099)